CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [Unreleased]
 
+## [1.4.0] - 2017-08-07
```

Comment:
1.4.0 was already released. Anyways it is preferable that you leave off the versioning to the maintainers as there is no guarantee on merge order or how long a review will take.